### PR TITLE
DateInterval: weeks and days can be combined since 8.0.0

### PR DESCRIPTION
**Pull request by saundefined**
Related issue #633

### CODE REVIEW

**Comment by kamil-tekiela** [comment]
Maybe also add to a changelog?

**Comment by saundefined** [comment]
@kamil-tekiela, sure, added. Thank you!

File: reference/datetime/dateinterval/construct.xml [hunk 1]
```diff
@@ -65,8 +65,8 @@
           <row>
            <entry><literal>W</literal></entry>
            <entry>
-            weeks. These get converted into days,
-            so can not be combined with <literal>D</literal>.
+            weeks. Converted into days.
```
Comment (nikic) [inline reference/datetime/dateinterval/construct.xml:68]: Does this "Converted into days" comment still make sense? I don't get what it's trying to say.

Comment (kamil-tekiela) [reply reference/datetime/dateinterval/construct.xml]: Yes, apparently this is how it works. If you use `P1W` then it gets converted into 7 days. If you use `P1W2D` then the days will take precedence and overwrite the value from weeks in PHP < 8

Comment (cmb69) [reply reference/datetime/dateinterval/construct.xml]: See https://3v4l.org/RNqK8; the object doesn't store weeks, but only days internally.

**Pull request merged**